### PR TITLE
Remove machine if exist on platform string

### LINKED ISSUE #14: Do not display `machine` if it is in `platform`
In the Markdown reports, we show `Platform: {platform} {machine}` in the summary.

However, some platforms already include the machine string, and so this is repetitive.

We should only display the machine string if `machine not in platform` is True.

### PR DESCRIPTION
**Pull request by eamanu**
Fix #14 

Remove the machine string if exist on platform string.

### CODE REVIEW

File: src/reportabug/__init__.py [hunk 1]
```diff
@@ -192,7 +192,13 @@ def format_markdown(data, github=False):
...
     print()
     print("* Python", data["platform"]["version"], data["platform"]["architecture"])
-    print("* Platform", data["platform"]["platform"], data["platform"]["machine"])
+
+    platform_info = (data["platform"]["platform"]
+                     if data["platform"]["machine"] in
+                     data["platform"]["platform"] else
+                     data["platform"]["platform"] + ' ' + data["platform"]["machine"])
+
+    print("* Platform %s " % (platform_info))
```
Comment (zooba) [inline src/reportabug/__init__.py]: I think this is more easy to read as a regular `if` block:

```python
if ...machine not in ...platform:
    print('* Platform', both)
else:
    print('* Platform', one)
```

**Comment by zooba** [comment]
And since we have tests, we should probably add tests for additional logic like this :) Though it probably needs a bit more infrastructure to make it handle well, so don't worry about it for now.

File: src/reportabug/__init__.py (hunk 1 above)
Comment (eamanu) [reply src/reportabug/__init__.py]: Hello  sorry for the delay. I send the commit. :-)

**Comment by zooba** [comment]
Great, thanks!

**Pull request merged**